src/Resources/app/storefront/src/js/filter-slider-range.plugin.js

```diff
@@ -318,4 +315,18 @@ export default class FilterSliderRange extends FilterBasePlugin {
       this.slider.noUiSlider.set([null, this._inputMax.value]);
     }
   }
+
+  refreshDisabledState(filter) {
```

Comment:
Is this method called by the parent? I can not see any caller in this class.

Reply:
Yes it is.